src/txn/txn_ckpt.c

```diff
@@ -745,8 +745,8 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
     WT_TXN_GLOBAL *txn_global;
     WT_TXN_ISOLATION saved_isolation;
     wt_timestamp_t ckpt_tmp_ts;
-    uint64_t finish_secs, hs_ckpt_duration_usecs, time_start_hs, time_stop_hs;
     uint64_t fsync_duration_usecs, generation, time_start_fsync, time_stop_fsync;
+    uint64_t hs_ckpt_duration_usecs, time_start_hs, time_stop_hs;
```

Comment:
Minor: I suggest moving the `hs_ckpt_duration_usecs` to the line above, and moving the 2 `time_*_fsync` to the 2nd line so all the `time_*` are on the same line together.